### DISCUSSION TITLE
WALLET AND CRM

### DISCUSSION THREAD

**Discussion by Wotan6891** [andrewdwallo/erpsaas#89] [Ideas]
Hello,

It would be a good idea:
1. A Wallet for internal transfers between users of the same company.
2. The email marketing function.
3. Mass sending to WhatsApp.
4. CRM.

Congratulations on your great work, and Regards

Carlos

**Comment by andrewdwallo** [comment]
Hi @Wotan6891,
Thanks for the ideas. Could you explain more about internal transfers idea? What would be the purpose (like could you give a scenario)?

**Reply by Wotan6891** [reply]
Hello,

I think that internal transfers would not fit well in an ERP.

I was wrong about this idea of ​​internal transfers hehe.

Greetings
